core/src/main/java/io/grpc/inprocess/InProcessTransport.java

```diff
@@ -31,7 +31,7 @@
 
 package io.grpc.inprocess;
 
-import static com.google.common.base.Preconditions.checkNotNull;
+import com.google.common.base.Preconditions;
```

Comment:
No. Generally I would leave it alone and have used the static checkNotNull here, but a previous change swapped to the static checkNotNull because the author of the change preferred that. I was against swapping back and forth needlessly but others didn't express any concern, so now I get to play the game.